pages/auth/login/callback/index.md

```diff
@@ -1,6 +0,0 @@
----
-title: Auth
-layout: page-react.html
-entryname: auth
-loadingMessage: Please wait while we log you in.
----
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-d9e419ac9838c0b3206697da30fea3d2R8.

This app does not have a dedicated E2E test, but it is part of the E2E login flow, so should be covered.